best_practices/controllers.rst

```diff
@@ -80,7 +80,7 @@ The ``@Template`` annotation is useful, but also involves some magic. We
 don't think its benefit is worth the magic, and so recommend against using
 it.
 
-Most of the time, ``@Template`` is used without any parameters, which makes
+Most of the time, ``@Template()`` is used without any parameters, which makes
```

Comment:
As I explained in another great PR of you, I don't like using `()` for annotations.